relay_test.go

```diff
@@ -565,6 +565,12 @@ func TestRelayRejectsDuringClose(t *testing.T) {
 		require.Error(t, err, "Expect call to fail after relay is shutdown")
 		assert.Contains(t, err.Error(), "incoming connection is not active")
 		close(block)
+		wg.Wait()
+		// FIXME:
+		// We fire all pending timers when we shut down the relay, which
+		// includes the timeout for the blocked handler. That unblocks the
+		// handler before we get to the call above, which obviates the point of
+		// this test.
```

Comment:
Maybe? I _think_ that's what this test was exercising. Separately, we weren't waiting on `wg` anywhere, but I think this is the right spot.